### PR TITLE
Add Library postgresql/basic as databases/postgresql

### PR DESCRIPTION
**Pull request by hhorak**
@lukaszachy @hegerj let's sync on the way it looks after the small re-org

The changes I did from the internal copy:

- Adding Vasek to authors as he's done some work recently in the code
- Rename the library from postgresql/basic to database/postgresql to match the new layout in https://github.com/beakerlib/database
- Bumping version to 4.0 as open-sourcing seems to be rather visible milestone

### CODE REVIEW

File: postgresql/runtest.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+#
+#   You should have received a copy of the GNU General Public
+#   License along with this program; if not, write to the Free
+#   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+#   Boston, MA 02110-1301, USA.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Include Beaker environment
+. /usr/bin/rhts-environment.sh || exit 1
```
Comment (hegerj) [inline postgresql/runtest.sh]: This line is obsolete and not needed.

**Review by hegerj — CHANGES_REQUESTED** [review]
There is no .fmf file for this library.
Also Makefile should not be upstreamed if not needed for some reason.

File: postgresql/runtest.sh (hunk 1 above)
Comment (hhorak) [reply postgresql/runtest.sh]: Thanks will remove.

Comment (hhorak) [reply postgresql/runtest.sh]: Done.

**Comment by hhorak** [comment]
Thanks for a quick response, first chunk of feedback should be addressed.

**Comment by hegerj** [comment]
Sorry I didn't express myself properly regarding the .fmf. 
fmf tree should be placed on root level of the repository (already is in this case) and each library should have its own .fmf file (usually main.fmf) with its metadata in the library directory.

**Comment by hhorak** [comment]
>  fmf tree should be placed on root level of the repository (already is in this case) and each library should have its own .fmf file (usually main.fmf) with its metadata in the library directory.

hups, need re-read the guidelines then, I kinda missed this. thanks for explaining.

File: postgresql/main.fmf [hunk 2]
```diff
@@ -0,0 +1,9 @@
+summary: Set of basic functions for postgresql
+contact:
+- Vaclav Danek <vdanek@redhat.com>
+test: ./runtest.sh
+framework: beakerlib
+recommend:
+- initscripts
+duration: 5m
+extra-task: /CoreOS/database/postgresql
```
Comment (hegerj) [inline postgresql/main.fmf]: I would remove these two attributes.
Duration doesn't really make sense with library and extra-task is for downstream.

File: postgresql/.fmf/version [hunk 3]
```diff
@@ -0,0 +1 @@
+1
```
Comment (hegerj) [inline postgresql/.fmf/version]: I don't think fmf tree inside library makes sense, I am not totaly sure though, @lukaszachy ?

File: postgresql/main.fmf (hunk 2 above)
Comment (hhorak) [reply postgresql/main.fmf]: Ok, will believe it doesn't make sense.

File: postgresql/.fmf/version (hunk 3 above)
Comment (hhorak) [reply postgresql/.fmf/version]: I agree, got to the same conclusion. Removed now.

**Review by hegerj — APPROVED** [review]
LGTM

**Comment by hhorak** [comment]
Rebased and tested in practice. With no changes from previous LGTM mark, I believe it's ready to merge.

File: postgresql/README.md [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+Jakub Prokes <jprokes@redhat.com>
+Vacla Danek <vdanek@redhat.com>
+
+# POD ERRORS
+
+Hey! **The above document had some coding errors, which are explained below:**
+
+- Around line 213:
+
+    You forgot a '=back' before '=head2'
```
Comment (psss) [inline postgresql/README.md:314]: Seems that you've documented some `pod` errors as well ;-)

Comment (hhorak) [reply postgresql/README.md:314]: Good catch, thanks!

Comment (hhorak) [reply postgresql/README.md:314]: Fixed.

**Pull request merged**